worksheet_A/TerminalHacking.cpp

```diff
@@ -39,8 +62,57 @@ int main()
 		std::cout << word << std::endl;
 	}
 
+	
 	// TODO: implement the rest of the game
+	
+	//Only runs loop while the player has lives left
+	while (lives > 0)
+	{
+		correctguess = false;
+		while (correctguess == false)
+		
+		{	
+			// Allows the player to enter an anwser
+			std::cout << "Please Enter Your Guess in CAPS\n";
+			
+			std::cin >> input;
+
+			// Checks all of the words in the list against the player's guess
+			for each (std::string word in options)
```

Comment:
`std::set` has a more efficient (logarithmic rather than linear) way of checking if it contains an element -- see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset